_posts/2016-06-01-welcome-to-magenta.markdown

```diff
@@ -111,7 +111,7 @@ images.
 
 This leads to perhaps our biggest challenge: combining generation,
 attention and surprise to tell a compelling story.  So much
-machine-generated music and art is good in small chunks, but lacks any
```

Comment:
I prefer to leave this comma. It's not strictly necessary, I agree. Please re-add.